FoundationsMastery/Child.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FoundationsMastery
+{
+    public class Child : Parent
+    {
+        public Child()
+        {
+            name = "Nathan Jr.";
+        }
+        public override void EyeColor()
+        {
+            base.EyeColor();
```

Comment:
Was attempting to show that the override was working by showing the base color as well